### PR TITLE
fix the aws serverless docs glitch

### PR DESCRIPTION
**Pull request by imnotashrimp**
# What changed

Cleaning up issues with the AWS serverless docs (cloudwatch, kinesis, guardduty)

----

## Pages to review

- [CloudWatch](https://deploy-preview-350--logz-docs.netlify.com/shipping/log-sources/cloudwatch.html)
- [GuardDuty](https://deploy-preview-350--logz-docs.netlify.com/shipping/log-sources/guardduty.html)
- [Kinesis](https://deploy-preview-350--logz-docs.netlify.com/shipping/log-sources/kinesis.html)

## Remaining work

- [x] Technical review
- [x] Copy Review

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- Update these log shipping pages in the app:
  - [ ] guardduty
  - [ ] cloudwatch
  - [ ] kinesis
- Update these readmes:
  - [ ] [CloudWatch readme](https://github.com/logzio/logzio_aws_serverless/blob/master/python3/cloudwatch/README.md)
  - [ ] [Kinesis readme](https://github.com/logzio/logzio_aws_serverless/blob/master/python3/kinesis/README.md)

### CODE REVIEW

File: _source/logzio_collections/_log-sources/cloudwatch.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ In the _Environment variables_ section, set your Logz.io account token, URL, and
...
 | URL <span class="required-param"></span> | Protocol, listener host, and port (for example, `https://<<LISTENER-HOST>>:8071`). <br > {% include log-shipping/replace-vars.html listener=true %} <!-- logzio-inject:listener-url --> |
-| TYPE <span class="required-param"></span> | The log type you'll use with this Lambda. This can be a [built-in log type]({{site.baseurl}}/user-guide/log-shipping/built-in-log-types.html), or a custom log type. <br> You should create a new Lambda for each log type you use. |
+| TYPE <span class="default-param">`logzio_cloudwatch_lambda`</span> | The log type you'll use with this Lambda. This can be a [built-in log type]({{site.baseurl}}/user-guide/log-shipping/built-in-log-types.html), or a custom log type. <br> You should create a new Lambda for each log type you use. |
 | FORMAT <span class="default-param">`text`</span> | `json` or `text`. If `json`, the Lambda function will attempt to parse the message field as JSON and populate the event data with the parsed fields. |
 | COMPRESS <span class="default-param">`false`</span> | Set to `true` to compress logs before sending them. Set to `false` to send uncompressed logs. |
```
Comment (idohalevi) [inline _source/logzio_collections/_log-sources/cloudwatch.md]: We're missing `ENRICH`: https://github.com/logzio/logzio_aws_serverless/tree/master/python2/cloudwatch

File: _source/logzio_collections/_log-sources/guardduty.md [hunk 2]
```diff
@@ -220,13 +220,10 @@ aws cloudformation deploy
...
-| LogzioURL <span class="default-param">`https://listener.logz.io:8071`</span> | {% include log-shipping/replace-vars.html listener='noReplace' %} <!-- logzio-inject:listener-url --> |
-| LogzioTYPE <span class="default-param">`logzio_kinesis_stream`</span> | The log type you'll use with this Lambda. This can be a [built-in log type]({{site.baseurl}}/user-guide/log-shipping/built-in-log-types.html), or a custom log type. <br> You should create a new Lambda for each log type you use. |
+| LogzioURL <span class="default-param">`https://listener.logz.io:8071`</span> | Protocol, listener host, and port (for example, `https://<<LISTENER-HOST>>:8071`). <br > {% include log-shipping/replace-vars.html listener='noReplace' %} <!-- logzio-inject:listener-url --> |
+| LogzioTYPE <span class="default-param">`logzio_cloudwatch_logs`</span> | The log type you'll use with this Lambda. This can be a [built-in log type]({{site.baseurl}}/user-guide/log-shipping/built-in-log-types.html), or a custom log type. <br> You should create a new Lambda for each log type you use. |
```
Comment (idohalevi) [inline _source/logzio_collections/_log-sources/guardduty.md]: Guardduty takes the logs from kinesis so the kinesis params should be here. You can understand it from step 2

File: _source/logzio_collections/_log-sources/kinesis.md [hunk 3]
```diff
@@ -76,7 +76,7 @@ In the _Environment variables_ section, set your Logz.io account token, URL, and
...
 | URL <span class="required-param"></span> | Protocol, listener host, and port (for example, `https://<<LISTENER-HOST>>:8071`). <br > {% include log-shipping/replace-vars.html listener=true %} <!-- logzio-inject:listener-url --> |
-| TYPE <span class="required-param"></span> | The log type you'll use with this Lambda. This can be a [built-in log type]({{site.baseurl}}/user-guide/log-shipping/built-in-log-types.html), or a custom log type. <br> You should create a new Lambda for each log type you use. |
+| TYPE <span class="default-param">`logzio_kinesis_stream`</span> | The log type you'll use with this Lambda. This can be a [built-in log type]({{site.baseurl}}/user-guide/log-shipping/built-in-log-types.html), or a custom log type. <br> You should create a new Lambda for each log type you use. |
 | FORMAT <span class="default-param">`text`</span> | `json` or `text`. If `json`, the Lambda function will attempt to parse the message field as JSON and populate the event data with the parsed fields. |
 | COMPRESS <span class="default-param">`false`</span> | Set to `true` to compress logs before sending them. Set to `false` to send uncompressed logs. |
```
Comment (idohalevi) [inline _source/logzio_collections/_log-sources/kinesis.md]: Again, where are the kinesis params? please check the PR in the repo or ask me if you don't understand the integration

File: _source/logzio_collections/_log-sources/cloudwatch.md (hunk 1 above)
Comment (imnotashrimp) [reply _source/logzio_collections/_log-sources/cloudwatch.md]: Is this also true for the cloudformation deployment?

Comment (idohalevi) [reply _source/logzio_collections/_log-sources/cloudwatch.md]: Indeed, great catch. @ronish31 can you please add it to the cloudformation template

Comment (ronish31) [reply _source/logzio_collections/_log-sources/cloudwatch.md]: yes

Comment (imnotashrimp) [reply _source/logzio_collections/_log-sources/cloudwatch.md]: @idohalevi I'm told `TYPE` is not required

**Pull request merged**